src/app/store.ts

```diff
@@ -1,10 +1,23 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
```

Comment:
pls refactor your code so that you won't have to disable `import/no-cycle` rule

Reply:
![image](https://github.com/mate-academy/react_redux-list-of-posts/assets/122940921/97d508e3-5244-47ad-b5c5-6c33b395d232)
![image](https://github.com/mate-academy/react_redux-list-of-posts/assets/122940921/d487e764-0b6f-4788-a082-6ca0664ace55)

Reply:
// eslint-disable-next-line import/no-cycle
so it was already given to me as example